mlflow/sklearn/__init__.py

```diff
@@ -740,17 +745,17 @@ def _log_pretraining_metadata(estimator, *args, **kwargs):  # pylint: disable=un
         # process; accordingly, we avoid logging initial, untuned parameters
         # for these seed estimators.
         should_log_params_deeply = not _is_parameter_search_estimator(estimator)
-        # Chunk model parameters to avoid hitting the log_batch API limit
-        for chunk in _chunk_dict(
-            estimator.get_params(deep=should_log_params_deeply),
-            chunk_size=MAX_PARAMS_TAGS_PER_BATCH,
-        ):
-            truncated = _truncate_dict(chunk, MAX_ENTITY_KEY_LENGTH, MAX_PARAM_VAL_LENGTH)
-            try_mlflow_log(mlflow.log_params, truncated)
-
-        try_mlflow_log(mlflow.set_tags, _get_estimator_info_tags(estimator))
```

Comment:
As part of this work, we should revisit the use of `try_mlflow_log`. As we begin to batch together logging operations, we must either find a way to permit certain operations to fail or revisit the UX / guarantees provided by autologging. As a strawman, I would argue that we should be less permissive with failed Tracking API calls and mark runs as `FAILED` when content cannot be logged successfully, as the absence of expected content may break user workflows or provide an incomplete record of training. I.e. `try_mlflow_log` seems to be used too liberally as-is.